src/test/e2e/e2e.ts

```diff
@@ -0,0 +1,43 @@
+import { Browser, Page } from 'puppeteer';
+
+const puppeteer = require('puppeteer');
+
+const port: number = parseInt(process.env.PORT, 10) || 8080;
+
+describe('End to end test', () => {
```

Comment:
Can we try a Page Object Model approach to our e2e? i think using a common testing approach like that would be 1) easier to read 2) easier to maintain

Reply:
the describe should be the journey we are completing so something closer to `finding a court or tribunal listing`

Reply:
> Can we try a Page Object Model approach to our e2e? i think using a common testing approach like that would be 1) easier to read 2) easier to maintain

or a Screenplay pattern? both would be useful